src/prepack-cli.js

```diff
@@ -111,12 +112,16 @@ function run(Object, Array, console, JSON, process, prepackStdin, prepackFileSyn
     },
     flags
   );
+  resolvedOptions.errorHandler =
+    (error: NativeError) => {
+      console.log(`${error.message}\n${error.stack}`);
```

Comment:
So the individual errors are logged here. Is this going to be a good experience? Will the end-user understand what's happening if the errors stem from some particular partial evaluation branch?